### ISSUE TITLE
registers.service.gov.uk broken links

### ISSUE THREAD

**Issue by rhiaro** [kanedata/find-that-charity#151]
Several of the source links on [the data sources page](https://findthatcharity.uk/about#data-sources) are to register.service.gov.uk, which seems to no longer be functional. Where is this data actually being pulled in from now? I think the local authorities might be coming from the [MySociety LA list](https://pages.mysociety.org/uk_local_authority_names_and_codes/datasets/uk_la_future/latest)? Not sure about the Cabinet Office Government organisations one.

**Comment by drkane** [comment]
Yes, annoyingly the registers project came to an end so we can't use them any more. You're right that this is a bit confusing on the data sources page though. I've made some changes so that the about page will show anything that's not been updated in a month as "archived".

In terms of the specific registers, yes we're using the mysociety list for the local authorities. I do have a replacement Gov.uk departments one which uses the gov.uk API and should be broadly compatible with the old one. But I haven't switched this on yet.

Thanks!

**Comment by drkane** [comment]
See also commit: d4f0e98d92d50d5615922eb7c51e872dcd325963 - might need to manually edit the source records for the ones that were updated
